src/cascadia/WpfTerminalControl/NativeMethods.cs

```diff
@@ -186,31 +186,31 @@ public enum SetWindowPosFlags : uint
         public static extern void TerminalSendOutput(IntPtr terminal, string lpdata);
 
         [DllImport("PublicTerminalCore.dll", CharSet = CharSet.Unicode, CallingConvention = CallingConvention.StdCall)]
-        public static extern uint TerminalTriggerResize(IntPtr terminal, short width, short height, out COORD dimensions);
+        public static extern uint TerminalTriggerResize(IntPtr terminal, int width, int height, out TilSize dimensions);

```

Comment:
These marshal as int32 _always_ on all platforms on all netfx builds, right?

Reply:
Yes, I had to check too: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types